vision/text/detect_words.py

```diff
@@ -71,7 +77,24 @@ def detect_russian_word(
                     self.tessdata["width"][i],
                     self.tessdata["height"][i],
                 )
-                verts = [(x, y), (x + w, y), (x, y + h), (x + w, y + h)]
+                ll_pt = (x, y + h)  # lower left point of rotated rectangle
+                ul_pt = (x, y)  # upper left point of rotated rectangle
+                ur_pt = (x + w, y)  # upper right point of rotated rectangle
+                lr_pt = (x + w, y + h)  # lower right point of rotated rectangle
+
+                # text detection is performed on a rotated image
+                # boxes need to be rotated back to their original position
+                rotated_verts = [ll_pt, ul_pt, ur_pt, lr_pt]
+
+                # rotate box back to original point on color image
+                rot_mat = cv2.getRotationMatrix2D(
+                    center=(x_ul, y_ul), angle=theta, scale=1
+                )
+                verts = cv2.transform(src=np.array([rotated_verts]), m=rot_mat)[0]
```

Comment:
> src=np.array([rotated_verts])

Instead of casting `rotated_verts` in the middle of this function call, you can do the cast when you initialize `rotated_verts` since it is not being used anywhere else.